web-wallet/src/routes/(app)/dashboard/+layout.svelte

```diff
@@ -56,6 +56,12 @@
     networkStatusIconPath = mdiLink;
     syncStatusLabel = `Dusk ${networkName}`;
   }
+
+  let networkName = "Unknown";
+
+  onMount(async () => {
```

Comment:
No need to wait for the component to mount: it's better to make the call immediately.

```js
networkStore.getNetworkName().then((name) => { networkName = name; }
```